### ISSUE TITLE
[Extensibility Bug] PageExtension OnLookUp not being executed anymore

### ISSUE THREAD

**Issue by bodenk** [microsoft/AL#7211]
Hello,

It looks like adding an OnLookUp() trigger in a pageextension to override the existing OnLookUp() on a page isn't working anymore from version 21.1 onwards. When we add a pageextension on for example "Item Tracking Lines" for "Package No." this code will not be executed.

```
pageextension 50100 "ItemTrackingLinesExtKBFDW" extends "Item Tracking Lines"
{
    layout
    {
        modify("Package No.")
        {
            trigger OnLookup(var Text: Text): Boolean
            begin
                Error('Execute some code here instead.');
            end;
        }
    }
}
```

In this snippet I just added an Error but is never triggered for 21.1, in version 21.0 we also tested this and it does work.

**Comment by thloke** [comment]
Hi, this issue is out of scope of for this repository. You should do one of the following:

- Open a support request to CSS through the PartnerSource portal
- Open a bug in Collaborate
- Contact your Service Account Manager (SAM) in your local subsidiary to understand what is included in your contract as of support incident and PAH (Partner Advisory Hours). Your SAM might also direct you "step by step" how to open a support request or how to get credentials, if this is the first time for you or your company

As a reminder, this repository only handles issues to do with the compiler or VSCode development experience for AL. I agree that this is a severe bug, but since **it's a runtime issue and not a language issue** it will need to go through support for it to reach the right team.

**Comment by thloke** [comment]
I know I closed this bug, but given the severity of this regression I've informally raised this with the team internally. A support case will give this more priority though.

**Comment by thloke** [comment]
Reopening, and accepting this issue.

**Comment by bodenk** [comment]
I see you accepted this issue now, if it is still required to open a support case please let us know and we'lll do so.

**Comment by JesperSchulz** [comment]
The fix for this issue has been checked in to the master branch. It will be available in the bcinsider.azurecr.io/bcsandbox-master Docker image starting from platform build number 22.0.47824.0 and VS Code Extension Version <None>. 
 
 If you don’t have access to these images you need to become part of the Ready2Go program: aka.ms/readytogo 
 
 For more details on code branches and docker images please read: 
 https://blogs.msdn.microsoft.com/nav/2018/05/03/al-developer-previews-multiple-releases-and-github/ 
 https://freddysblog.com/2020/06/25/working-with-artifacts/

**Comment by bodenk** [comment]
Will this be backported to version 21.1? 
We're experiencing this issue for Lot No. as well.

**Comment by thloke** [comment]
Yes, it has been backported to our 21 branch, so it should be in 21.1.

**Comment by bodenk** [comment]
Thank you, I will check again. I tested this week on version 21.1.47986 but was still failing.

**Comment by thloke** [comment]
The change just went in this Monday. I think the version that has it is from 21.1.48039
